packages/react-events/src/Press.js

```diff
@@ -402,12 +401,36 @@ function calculateResponderRegion(target: Element, props: PressProps) {
   };
 }
 
+function isTouchEvent(nativeEvent: Event): boolean {
+  return Array.isArray((nativeEvent: any).changedTouches);
+}
+
+function getEventPageCoords(
+  nativeEvent: Event,
+): {x: null | number, y: null | number} {
+  let eventObject = (nativeEvent: any);
+  if (isTouchEvent(eventObject)) {
+    const touchEvents = eventObject.changedTouches;
+    if (touchEvents.length === 0) {
+      throw new Error('TouchEvent does not have any changed touches');
+    }
+    // TODO: we need to support multi-touch in the future, this is a temp
```

Comment:
How does multi-touch related to Press?

Reply:
As we discussed in our meeting, we need to consider the user pressing multiple Press event responders at the same time. I'll address this TODO tomorrow in a follow up.